core/jobs/transforms/validation/user_validation.py

```diff
@@ -113,33 +113,6 @@ def process(
                 model, incorrect_keys)
 
 
-# TODO(#15613): Here we use MyPy ignore because the incomplete typing of
-# apache_beam library and absences of stubs in Typeshed, forces MyPy to
-# assume that DoFn class is of type Any. Thus to avoid MyPy's error (Class
-# cannot subclass 'DoFn' (has type 'Any')), we added an ignore here.
-@validation_decorators.AuditsExisting(user_models.UserQueryModel)
-class ValidateOldModelsMarkedDeleted(beam.DoFn):  # type: ignore[misc]
```

Comment:
Shouldn't there be some corresponding test elsewhere for the more general functionality?

Reply:
Not sure what you mean.

Reply:
Sorry, ignore me. I misread this as a test.

One question though. Why are we deleting this? Is it just unused? Do you remember why we added it originally?

Reply:
Unused. It was some old preparation for the audit jobs, but it is no longer relevant and the last design of the audit job doesn't expect them.